src/IoTHub.Portal.Shared/Models/v1.0/AccessControlDto.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) CGI France. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace IoTHub.Portal.Shared.Models.v1._0
```

Comment:
Do not name Dto with suffix DTO.
Change the namespace to use : 
```suggestion
namespace IoTHub.Portal.Shared.Models.v10
```